test/tools/jsontests/TransactionTests.cpp

```diff
@@ -85,6 +85,8 @@ void doTransactionTests(json_spirit::mValue& _v, bool _fillin)
 				//Transaction is InValid
 				cnote << "Transaction Exception: " << diagnostic_information(_e);
 				o.erase(o.find("transaction"));
+				if (o.count("sender") > 0)
```

Comment:
Not sure I understand what do these changes do. It influences the generated json files?